assets/javascripts/discourse/components/discourse-post-event/invitee.gjs

```diff
@@ -0,0 +1,65 @@
+import Component from "@glimmer/component";
+import { concat } from "@ember/helper";
+import { service } from "@ember/service";
+import { htmlSafe } from "@ember/template";
+import { eq } from "truth-helpers";
+import AvatarFlair from "discourse/components/avatar-flair";
+import concatClass from "discourse/helpers/concat-class";
+import { formatUsername } from "discourse/lib/utilities";
+import { avatarImg } from "discourse-common/lib/avatar-utils";
+
+export default class DiscoursePostEventInvitee extends Component {
+  @service site;
+  @service currentUser;
+
+  get statusIcon() {
+    let statusIcon;
+    switch (this.args.invitee.status) {
+      case "going":
+        statusIcon = "fa-check";
+        break;
+      case "interested":
+        statusIcon = "fa-star";
+        break;
+      case "not_going":
+        statusIcon = "fa-times";
+        break;
+    }
+    return statusIcon;
+  }
+
+  get avatarImage() {
+    return htmlSafe(
+      avatarImg({
+        avatarTemplate: this.args.invitee.user.avatar_template,
+        size: this.site.mobileView ? "tiny" : "large",
+        title: this.args.invitee.user.name
+          ? formatUsername(this.args.invitee.user.name)
+          : this.args.invitee.user.username,
+      })
+    );
+  }
+
+  <template>
+    <li
+      class={{concatClass
+        "event-invitee"
+        (if @invitee.status (concat "status-" @invitee.status))
+        (if (eq this.currentUser.id @invitee.user.id) "is-current-user")
+      }}
+    >
+      <a class="topic-invitee-avatar" data-user-card={{@invitee.user.username}}>
+        {{this.avatarImage}}
+        {{#if this.statusIcon}}
+          <AvatarFlair
+            @flairName={{concat
+              "discourse_calendar.discourse_post_event.models.invitee.status."
```

Comment:
the widget code is convoluted but I think this string eventually ends up there:

https://github.com/discourse/discourse/blob/main/app/assets/javascripts/discourse/app/widgets/widget.js#L511

so it should be an i18n path

Reply:
Oh, TIL! I was comparing with [old widget](https://github.com/discourse/discourse-calendar/blob/54460416c2efe30f8265ed7b5aa06743f615e8b3/assets/javascripts/discourse/widgets/discourse-post-event-invitee.js#L54-L56) implementation and it didn't even cross my mind that it could be handled by the widget system itself instead of the avatar flair widget 😄 Thank you, resolving this!